app/form_models/publishers/job_listing/application_form_form.rb

```diff
@@ -0,0 +1,62 @@
+class Publishers::JobListing::ApplicationFormForm < Publishers::JobListing::UploadBaseForm
+  CONTENT_TYPES_ALLOWED = %w[
+    application/pdf
+    application/msword
+    application/vnd.openxmlformats-officedocument.wordprocessingml.document
+  ].freeze
+
+  validates :application_email, presence: true
+  validate :other_application_email_presence
+  validate :application_form_presence
+
+  def self.fields
+    %i[application_email]
+  end
+  attr_accessor(:application_form, *fields)
+  attr_writer(:other_application_email)
+
+  def valid_application_form
+    @valid_application_form ||= application_form if application_form &&
+                                                    valid_file_size?(application_form) &&
+                                                    valid_file_type?(application_form) &&
+                                                    virus_free?(application_form)
+  end
+
+  def file_upload_field_name
+    :application_form
+  end
+
+  def application_email
+    return unless @vacancy.application_email || params[:application_email]
+
+    if @vacancy.application_email == @current_publisher&.email
+      @current_publisher.email
+    else
+      "other"
+    end
+  end
+
+  def other_application_email
+    @vacancy.application_email unless @vacancy.application_email == @current_publisher&.email
+  end
+
+  def params_to_save
+    {
+      application_email: params[:application_email] == "other" ? params[:other_application_email] : params[:application_email],
+    }
+  end
+
+  private
+
+  def content_types_allowed
```

Comment:
is this needed? i cant see the method name `content_types_allowed` is used anywhere